userprofile/models.py

```diff
@@ -31,12 +31,34 @@
 
 # Models ######################################################################
 
+
+class Organization(ValidateModelMixin, TimeStampedModel):
+    """
+    Organizations model
+    """
+    name = models.CharField(max_length=255)
+    github_handle = models.CharField(unique=True, max_length=255)
```

Comment:
@rocioar This field isn't used anywhere at the moment, so I think it would be fine to make it optional. Right now I'm thinking that we'll mostly be using it for informational purposes (and won't strictly need it to be able to implement the first batch of access control and billing features). But that might change in the future.

On a related note, it would be good to establish a direct connection between watched forks and organizations (by changing `WatchedFork.organization` to point to the `Organization` model), to prepare for implementing the billing features. Right now, the only way for us to relate an organization to a watched fork would be via the `WatchedFork.organization` and `Organization.github_handle` fields. But that will break down if an organization (a/k/a a client) would like to use PR watch functionality for multiple GitHub organizations.

Reply:
I made the field optional.

Make sense to update the WatchedFork to point to the new organization model, I will update it.